c/cert/test/rules/FIO38-C/test.c.qcc

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+// Workaround for the Musl implementing FILE as an incomplete type.
+#if !defined(__DEFINED_struct__IO_FILE)
+struct _IO_FILE {
+  char __x;
+};
+#define __DEFINED_struct__IO_FILE
+#endif
+
+int f1(void) {
+  FILE my_stdout = *stdout; // NON_COMPLIANT
+  return fputs("Hello, World!\n", &my_stdout);
+}
+
+int f2(void) {
+  FILE *my_stdout;
+  my_stdout = stdout;           // COMPLIANT
+  FILE my_stdout2 = *my_stdout; // NON_COMPLIANT
+  return fputs("Hello, World!\n", my_stdout);
+}
+int f2b(void) {
+  FILE *const *my_stdout;
+  // my_stdout = &stdout;        // does not compile in qcc
```

Comment:
Is this legal C? Or is it that case that you are not guaranteed to be able to take the address of `stdout`?